camelcase.go

```diff
@@ -7,28 +7,17 @@ import "unicode"
 // Split splits the camelcase word and returns a list of words. It also
 // supports digits.  Both lower camel case and upper camel case are supported.
 // For more info please check:  http://en.wikipedia.org/wiki/CamelCase
-//
-// Below are some example cases:
-//   lowercase =>       ["lowercase"]
-//   Class =>           ["Class"]
-//   MyClass =>         ["My", "Class"]
-//   MyC =>             ["My", "C"]
-//   HTML =>            ["HTML"]
-//   PDFLoader =>       ["PDF", "Loader"]
-//   AString =>         ["A", "String"]
-//   SimpleXMLParser => ["Simple", "XML", "Parser"]
-//   vimRPCPlugin =>    ["vim", "RPC", "Plugin"]
-//   GL11Version =>     ["GL", "11", "Version"]
-//   99Bottles =>       ["99", "Bottles"]
-//   May5 =>            ["May", "5"]
-//   BFG9000 =>         ["BFG", "9000"]
```

Comment:
Thanks Peter for the fix! Much appreciated. I think we should not remove this as it shows simply in the first place what's going on and what the package does. +1 for adding it to the godoc, but sometimes people just discover this from Github, and not godoc.org, so there is still the possibility they can't see it without clicking on the godoc link.